app/helpers/application_helper.rb

```diff
@@ -1,2 +1,16 @@
 module ApplicationHelper
+  def area_tag(area, classes = [])
```

Comment:
You can make use of variable number of arguments here, as in: `def area_tag(area, *classes)`. Classes will be an array by default :)